File: java & sql/day05/src/com/pm/Ex10.java
package com.pm;

public class Ex10 {
//	public static int su1=1111;
	public int su2 = 2222;

	public static void main(String[] args) {
//		int su1=1111;
//		System.out.println("main su1="+su1);
//		func01(su1);
//		System.out.println("main su1="+su1);
		Ex10 me = new Ex10();
		System.out.println("main su2 = " + me.su2);
		me.func02();
		Ex10 me2 = new Ex10();
		System.out.println("main su2 = " + me2.su2);
		System.out.println("main su2 = " + me.su2);
	}

	public void func02() {
		su2++;
		System.out.println("f2 su2 = " + su2);
	}

	public static void func01(int su1) {
		su1++;
		System.out.println("func01 su1 = " + su1);
	}
}

File: stsWorkspace/node01/ex10.js
function Func01(){this.su1 = 1111;}

console.log('Func01.su1 = '+new Func01().su1);

class Fun02{
    su1=1111;
    func1=function(){console.log('메서드',this);};
    func3=()=>{console.log('메서드',this);};
    func2(a){console.log('메서드',a,this);};
    constructor(a){
        this.su2=a;
    }
}
// console.log(new Fun02(3333));
new Fun02(3333).func2(1111);